ArtOfIllusion/src/artofillusion/RotateViewTool.java

```diff
@@ -208,13 +214,13 @@ private void dragRotateSpace(WidgetMouseEvent e, ViewerCanvas view)
 		dx = dragPoint.x-clickPoint.x;
 		dy = dragPoint.y-clickPoint.y;
 
-		if (controlDown && !e.isShiftDown())
+		if (mouseButtonTwo(e) && controlDown)
 		{
 			view.tilting = true;
 			tilt(e, view, clickPoint);
 			return;
 		}
-		else if (controlDown && e.isShiftDown()){
+		else if (mouseButtonThree(e) && e.isShiftDown()){
```

Comment:
No MB3 available here!

Instead you need to know that control is down. This is the temporary switch from rotate to pan, by pressing both, the control and shift keys.

Reply:
Yep...